ui/app/components/app/gas-customization/gas-modal-page-container/index.scss

```diff
@@ -41,10 +41,10 @@
     }
 
     &__title {
+      @include H5;
+
       color: $black;
-      font-size: 16px;
       font-weight: 500;
-      line-height: 16px;
```

Comment:
Note that this does make the header of the "Customize gas" page 6px taller, but... I think it looks better personally 🤷

Reply:
Before:

![customize-gas-before](https://user-images.githubusercontent.com/2459287/89460906-7ab8d100-d741-11ea-903a-a1611f50687f.png)

After:

![customize-gas-after](https://user-images.githubusercontent.com/2459287/89460913-7ee4ee80-d741-11ea-9e47-9bde440451a0.png)

Reply:
I agree, i like the "after"